db/1.9/modules/ROOT/pages/database-connector-transactions.adoc

```diff
@@ -0,0 +1,78 @@
+= Configure Database Transactions Example - Mule 4
+
+Anypoint Connector for Database (Database Connector) enables you to execute database operations in the context of a transaction. Each operation has a Transactional Action field where you specify the type of joining action that operations can take regarding the active transaction.
+
+The following examples illustrates how to configure the Transactional Action field in Studio both for the Select operation, and when grouping operations using the Try scope component.
+
+== Configure the Transactional Action Field in Studio
+
+To configure the Transactional Action field for the Select operation follow the next steps:
+
+. In your Studio flow, select the *Select* operation.
+. In the connector configuration screen, click the *Advanced* tab.
+. Set the *Transactional action* field to any of the following options:
```

Comment:
Note capitalization and formatting for "transactional action field." 
I think it would be okay to use this in the narrative preceding, although naming it instead of "presenting" its GUI appearance is OK too.